src/renderer/components/cluster-manager/lens-views.ts

```diff
@@ -20,10 +20,9 @@ export async function initView(clusterId: ClusterId) {
     return;
   }
   logger.info(`[LENS-VIEW]: init dashboard, clusterId=${clusterId}`)
-  const cluster = clusterStore.getById(clusterId);
   const parentElem = document.getElementById("lens-views");
   const iframe = document.createElement("iframe");
-  iframe.name = cluster.contextName;
+  iframe.name = clusterId;
```

Comment:
this line is causing integration tests to fail:
```
const waitForMinikubeDashboard = async (app: Application) => {
    ...
    await app.client.waitForExist(`iframe[name="minikube"]`)
    ...
}
```

Reply:
Right, will fix.